commands/displayers/database.go

```diff
@@ -59,34 +60,37 @@ func (d *Databases) Cols() []string {
 		"Size",
 		"URI",
 		"Created",
+		"StorageMib",
 	}
 }
 
 func (d *Databases) ColMap() map[string]string {
 	if d.Short {
 		return map[string]string{
-			"ID":       "ID",
-			"Name":     "Name",
-			"Engine":   "Engine",
-			"Version":  "Version",
-			"NumNodes": "Number of Nodes",
-			"Region":   "Region",
-			"Status":   "Status",
-			"Size":     "Size",
+			"ID":         "ID",
+			"Name":       "Name",
+			"Engine":     "Engine",
+			"Version":    "Version",
+			"NumNodes":   "Number of Nodes",
+			"Region":     "Region",
+			"Status":     "Status",
+			"Size":       "Size",
+			"StorageMib": "StorageMib",
 		}
 	}
 
 	return map[string]string{
-		"ID":       "ID",
-		"Name":     "Name",
-		"Engine":   "Engine",
-		"Version":  "Version",
-		"NumNodes": "Number of Nodes",
-		"Region":   "Region",
-		"Status":   "Status",
-		"Size":     "Size",
-		"URI":      "URI",
-		"Created":  "Created At",
+		"ID":         "ID",
+		"Name":       "Name",
+		"Engine":     "Engine",
+		"Version":    "Version",
+		"NumNodes":   "Number of Nodes",
+		"Region":     "Region",
+		"Status":     "Status",
+		"Size":       "Size",
+		"StorageMib": "Storage Size - MiB",
```

Comment:
I wonder if we could truncate this just a bit? This table is already a bit big for most terminals, so saving a few characters might be nice. 

```suggestion
		"StorageMib": "Storage (MiB)",
```

How does that look to you?

Reply:
this looks good to me, thanks for the additional context on matching short/long versions. I made the changes @andrewsomething